mentorship ios/Models/ProfileModel.swift

```diff
@@ -0,0 +1,71 @@
+//
+//  ProfileModel.swift
+//  mentorship ios
+//
+//  Created by Yugantar Jain on 12/06/20.
+//  Copyright © 2020 Yugantar Jain. All rights reserved.
```

Comment:
can we remove this?

Reply:
@sunjunkie Why this in happening even after adding custom file header?